inst/bslib-scss/spacer.scss

```diff
@@ -13,14 +13,19 @@ $spacer: 1rem !default;
 // need a way to reset those margin-bottom to 0 in those special contexts
 //
 // We do this by adding this class to components (e.g., card())...
-.bslib-mb-spacer {
+.bslib-mb-spacing {
   margin-bottom: var(--bslib-mb-spacer);
 }
 
 // ...And this class for layout containers (e.g, layout_columns())
 .bslib-gap-spacing {
   gap: var(--bslib-mb-spacer);
-  > .bslib-mb-spacer, > .form-group, > p, > pre {
+  > .bslib-mb-spacing, > .form-group, > p, > pre {
     margin-bottom: 0;
   }
 }
+
+// Remove bslib margin bottom spacing for fill items in a fillable container
+.html-fill-container > .html-fill-item.bslib-mb-spacing {
+  margin-bottom: 0;
+}
```

Comment:
Just curious, do you have a concrete example of where this is relevant?

Reply:
I was thinking of something like `page_fillable(layout_columns())` or `card(layout_columns())`. I haven't tested it, but I'd expect there to be unwanted bottom margin if `layout_columns` has `.bslib-mb-spacing` but it's a fill item in a fillable container (where no spacing is wanted).

OTOH, those situations are probably covered by `.bslib-gap-spacing`.

Reply:
Ok, how about we file an issue about this and consider it for later? If there is a motivation for extending the spacing concept more generically, there might be more to consider here

Reply:
Sounds good to me!

```suggestion
```